Nimble Templates/App.xctemplate/TemplateInfo.plist

```diff
@@ -1,215 +1,391 @@
+					<string>___VARIABLE_bundleIdentifierPrefix:bundleIdentifier___.___PACKAGENAMEASRFC1034IDENTIFIER___.uat</string>
+				</dict>
+				<key>UAT</key>
+				<dict>
+					<key>ENABLE_NS_ASSERTIONS</key>
+					<string>NO</string>
+					<key>GCC_PREPROCESSOR_DEFINITIONS</key>
+					<string>UAT=1 $(inherited)</string>
+					<key>MTL_ENABLE_DEBUG_INFO</key>
+					<string>NO</string>
+					<key>SWIFT_COMPILATION_MODE</key>
+					<string>wholemodule</string>
+					<key>GCC_OPTIMIZATION_LEVEL</key>
+					<string>0</string>
+					<key>SWIFT_OPTIMIZATION_LEVEL</key>
+					<string>-Onone</string>
+					<key>SWIFT_ACTIVE_COMPILATION_CONDITIONS</key>
+					<string>UAT</string>
+					<key>DEBUG_INFORMATION_FORMAT</key>
+					<string>dwarf-with-dsym</string>
+					<key>PRODUCT_BUNDLE_IDENTIFIER</key>
+					<string>___VARIABLE_bundleIdentifierPrefix:bundleIdentifier___.___PACKAGENAMEASRFC1034IDENTIFIER___.uat</string>
+				</dict>
+				<key>Dev Production</key>
+				<dict>
+					<key>ONLY_ACTIVE_ARCH</key>
+					<string>YES</string>
+					<key>GCC_OPTIMIZATION_LEVEL</key>
+					<string>s</string>
+					<key>GCC_PREPROCESSOR_DEFINITIONS</key>
+					<string>DEBUG=1 PRODUCTION=1 $(inherited)</string>
+					<key>MTL_ENABLE_DEBUG_INFO</key>
+					<string>INCLUDE_SOURCE</string>
+					<key>ENABLE_TESTABILITY</key>
+					<string>YES</string>
+					<key>GCC_DYNAMIC_NO_PIC</key>
+					<string>NO</string>
+					<key>SWIFT_OPTIMIZATION_LEVEL</key>
+					<string>-O</string>
+					<key>SWIFT_ACTIVE_COMPILATION_CONDITIONS</key>
+					<array>
+						<string>DEBUG</string>
+						<string>PRODUCTION</string>
+					</array>
+					<key>DEBUG_INFORMATION_FORMAT</key>
+					<string>dwarf</string>
+					<key>PRODUCT_BUNDLE_IDENTIFIER</key>
+					<string>___VARIABLE_bundleIdentifierPrefix:bundleIdentifier___.___PACKAGENAMEASRFC1034IDENTIFIER___</string>
+				</dict>
+				<key>Production</key>
+				<dict>
+					<key>ENABLE_NS_ASSERTIONS</key>
+					<string>NO</string>
+					<key>GCC_PREPROCESSOR_DEFINITIONS</key>
+					<string>PRODUCTION=1 $(inherited)</string>
+					<key>MTL_ENABLE_DEBUG_INFO</key>
+					<string>NO</string>
+					<key>GCC_OPTIMIZATION_LEVEL</key>
+					<string>s</string>
+					<key>SWIFT_COMPILATION_MODE</key>
+					<string>wholemodule</string>
+					<key>SWIFT_OPTIMIZATION_LEVEL</key>
+					<string>-O</string>
+					<key>SWIFT_ACTIVE_COMPILATION_CONDITIONS</key>
+					<string>PRODUCTION</string>
+					<key>DEBUG_INFORMATION_FORMAT</key>
+					<string>dwarf-with-dsym</string>
+					<key>PRODUCT_BUNDLE_IDENTIFIER</key>
+					<string>___VARIABLE_bundleIdentifierPrefix:bundleIdentifier___.___PACKAGENAMEASRFC1034IDENTIFIER___</string>
+				</dict>
+			</dict>
+		</dict>
+		<key>Targets</key>
+		<array>
+			<dict>
+				<key>Name</key>
+				<string>___PACKAGENAME___</string>
+				<key>SharedSettings</key>
+				<dict>
+					<key>PRODUCT_NAME</key>
+					<string>$(TARGET_NAME)</string>
+					<key>SWIFT_VERSION</key>
+					<string>5.0</string>
```

Comment:
Please check the version here, when using the Xcode 12 to create a new project, what is the version here?

Reply:
It is 5.0 on Xcode 12